NorthstarDedicatedTest/logging.cpp

```diff
@@ -407,6 +407,86 @@ bool CClientState_ProcessPrint_Hook(__int64 thisptr, __int64 msg)
 	return true;
 }
 
+
+typedef double(__fastcall* GetTimeConnectedType)(__int64 thisptr);
+GetTimeConnectedType GetTimeConnected;
+
+typedef char*(__fastcall* GetAddressType)(__int64 thisptr);
+GetAddressType GetAddress;
+
+const char* COM_FormatSeconds(int seconds)
+{
+	static char string[64];
```

Comment:
Would need @pg9182's opinion her on best practices and all but passing string pointer to function instead of allocating a `static` on heap would probably be preferred.

Reply:
This function is from the CS:GO source leak.